BookPlayer.xcodeproj/project.pbxproj

```diff
@@ -358,6 +358,9 @@
 		C3FA301E20E0024900393DDA /* BPArtworkView.swift in Sources */ = {isa = PBXBuildFile; fileRef = C3FA301D20E0024900393DDA /* BPArtworkView.swift */; };
 		C3FE3F8220A090880055B9C6 /* limitPanAngle.swift in Sources */ = {isa = PBXBuildFile; fileRef = C3FE3F8120A090880055B9C6 /* limitPanAngle.swift */; };
 		C3FE94792080086800BCEA37 /* BookCellView.swift in Sources */ = {isa = PBXBuildFile; fileRef = C3FE94782080086800BCEA37 /* BookCellView.swift */; };
+		E9F4CFF6278AADD1003DBC4C /* SpeedCoordinator.swift in Sources */ = {isa = PBXBuildFile; fileRef = E9F4CFF5278AADD1003DBC4C /* SpeedCoordinator.swift */; };
+		E9F4CFF9278AAE7D003DBC4C /* SpeedViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = E9F4CFF7278AAE7D003DBC4C /* SpeedViewController.swift */; };
```

Comment:
It looks like this new file was not committed since it's not visible in the diff 👀

Reply:
strange. git didn't add this file, updated